--- /tmp/vi-orig/printf.c	2000-05-22 01:28:57
+++ printf.c	2026-07-03 21:48:42
@@ -11,7 +11,7 @@
 static char *printf_id = "@(#) printf.c:2.2 6/5/79";
 #endif
 
-#include "varargs.h"
+#include <stdarg.h>
 /*
  * This version of printf is compatible with the Version 7 C
  * printf. The differences are only minor except that this
@@ -30,8 +30,7 @@
 
 char *_p_dconv();
 
-printf(va_alist)
-	va_dcl
+printf(char *fmt0, ...)
 {
 	va_list ap;
 	register char *fmt;
@@ -43,7 +42,8 @@
 	char *ptr;
 	char buf[134];
 
-	va_start(ap);
+	va_start(ap, fmt0);
+	fmt = fmt0;
 	fmt = va_arg(ap,char *);
 	for (;;) {
 		/* process format string first */
